docs/tutorials/build/oauth.md

```diff
@@ -0,0 +1,573 @@
+
+    </div>
+    <div markdown>
+
+    <a href="https://developer.todoist.com/guides/#step-1-authorization-request">
+      <img src="../../../images/tutorial_oauth_scope_comma.png" srcset="../../../images/tutorial_oauth_scope_comma_2x.png 2x" class="screenshot" alt="The documentation for the scope separator.">
+    </a>
+
+    </div>
+    </section>
+
+
+## :material-application-edit: Register your application
+
+Every application needs to identify itself during the OAuth2 flow, so that the user knows who they are granting access to. This is done using a client ID and secret, which are provided when you register your application with the API provider.
+
+Most API providers host a developer console for creating and managing applications, and you should consult the API documentation to determine the exact steps required.
+
+=== ":material-numeric-1-circle: Open console"
+
+    <section class="tutorial-row" markdown>
+    <div markdown>
+
+    Open the Todoist [App Management Console][todoist_console] and sign in with your Todoist account.
+
+    Click the **Create a new app** button to start the process of registering an application.
+
+    </div>
+    <div markdown>
+
+    <img src="../../../images/tutorial_oauth_console.png" srcset="../../../images/tutorial_oauth_console_2x.png 2x" class="screenshot" alt="The Todoist app management console">
+
+    </div>
+    </section>
+
+
+=== ":material-numeric-2-circle: Create application"
+
+    <section class="tutorial-row" markdown>
+    <div markdown>
+
+    Enter a name for your application in the **App name** field. This will be shown later to the user when they are approving access.
+
+    For the **App service URL** enter `https://coda.io` for now. Once your Pack is complete, change it to the URL of the listing page for your Pack.
+
+    Click the **Create app** button.
+
+    </div>
+    <div markdown>
+
+    <img src="../../../images/tutorial_oauth_create.png" srcset="../../../images/tutorial_oauth_create_2x.png 2x" class="screenshot" alt="Creating a new application in the console">
+
+    </div>
+    </section>
+
+
+=== ":material-numeric-3-circle: Edit settings"
+
+    <section class="tutorial-row" markdown>
+    <div markdown>
+
+    The settings screen for your application includes the client ID and secret you'll need during the OAuth2 flow. You'll have to enter those into the Pack Studio later.
+
+    It's important to set the **OAuth redirect URL** field. This is the URL that the user's browser is sent to after they approve access. All packs use the same value for this URL:
+
+    ```
+    https://coda.io/packsAuth/oauth2
+    ```
+
+    Click the **Save settings** button to save your changes.
+
+    </div>
+    <div markdown>
+
+    <img src="../../../images/tutorial_oauth_settings.png" srcset="../../../images/tutorial_oauth_settings_2x.png 2x" class="screenshot" alt="The settings for the new application">
```

Comment:
I assume the client id and secret in the screenshot have been reset so that they're no longer usable?

Reply:
It is now! Was planning to, but hadn't remembered until this reminder.